data/src/main/java/io/gnosis/data/repositories/TokenRepository.kt

```diff
@@ -23,6 +24,27 @@ class TokenRepository(
                 Balance(token, it.balance, it.balanceUsd)
             }
 
+    //FIXME: use client gateway (grouping and sorting will be done on the backend side)
```

Comment:
Do we already have a ticket for this that we could mention here to check whether this can be remocved?